corehq/util/es/interface.py

```diff
@@ -113,6 +115,26 @@ def scroll(self, scroll_id=None, body=None, params=None, **kwargs):
         return results
 
     def scan(self, index_alias, query, doc_type):
+        """Call the elasticsearch-py library's `scan()` helper function.
+
+        In library versions <5.x this results in a `search_type='scan'` query.
+        In library versions >=5.x this results in the query['sort'] getting
+        silently clobbered to '_doc'.
+
+        To handle this appropriately across versions >=2.x:
```

Comment:
Would it be more correct to say `>=2.1`? (or whatever version >2.0 where `search_type='scan'` was introduced)